io/zenoh-links/zenoh-link-tls/src/unicast.rs

```diff
@@ -603,6 +604,18 @@ impl TlsClientConfig {
             client_auth = value.parse()?
         }
 
+        let server_name_verification: bool =
+            if let Some(value) = config.get(TLS_SERVER_NAME_VERIFICATION) {
+                value
+            } else {
+                TLS_SERVER_NAME_VERIFICATION_DEFAULT
+            }
+            .parse()?;
```

Comment:
This could be written more shortly:
```Rust
let server_name_verification: bool = config.get(TLS_SERVER_NAME_VERIFICATION)
  .unwrap_or(TLS_SERVER_NAME_VERIFICATION_DEFAULT).parse()?;
```